lib/fog/aws/models/compute/security_group.rb

```diff
@@ -327,9 +327,15 @@ def fetch_ip_permission(range, options)
           }
 
           if options[:group].nil?
-            ip_permission['IpRanges'] = [
-              { 'CidrIp' => options[:cidr_ip] || '0.0.0.0/0' }
-            ]
+            if options[:cidr_ipv6].nil?
```

Comment:
Are the `CidrIp` and `CidrIpv6` mutually exclusive? This implementation appears to reflect that, so I wanted to check-in in case it was possible to do dual stack type stuff. Does that make sense?

Reply:
I could not create a permission with both `CidrIp` and `CidrIpv6`.
With a request body like:
```
Action=AuthorizeSecurityGroupIngress&GroupId=sg-xxxxxxx&IpPermissions.1.FromPort=443&IpPermissions.1.IpProtocol=tcp&IpPermissions.1.IpRanges.1.CidrIp=1.2.3.4%2F0&IpPermissions.1.Ipv6Ranges.1.CidrIpv6=xxxx%3xxxx%3Axxxx%3Axxxx%3A%3A%2F64&IpPermissions.1.ToPort=443&Version=2016-11-15
```

It would return an error like:
```xml
<Code>InvalidPermission.Duplicate</Code><Message>the specified rule \\"peer: 0.0.0.0/0, TCP, from port: 443, to port: 443, ALLOW\\" already exists</Message></Error>
```
It is also not supported in the AWS web console.

Reply:
Thanks for confirming, I suspected something like that might be the case, but wanted to double check in case we might be missing something.